### PR TITLE
ninja code

### PR DESCRIPTION
**Pull request by ayushhagarwal**
Creating pull request for translation of ninja code section. Will review it once again myself and other other reviews and comments are welcomed.

### CODE REVIEW

File: 1-js/03-code-quality/04-ninja-code/article.md [hunk 1]
```diff
@@ -1,147 +1,147 @@
-# Ninja code
+# निंजा कोड
 
 
-```quote author="Confucius (Analects)"
-Learning without thought is labor lost; thought without learning is perilous.
+```उद्धरण लेखक = "कन्फ्यूशियस (अनालेक्ट्स)"
```
Comment (theBstar) [inline 1-js/03-code-quality/04-ninja-code/article.md:4]: These are smart tags, used by the page generator to generate the doc site. The site generator will fail if we change them.

**Review by theBstar — COMMENTED** [review]
Thanks for the MR @ayushhagarwal. I can see a lot of potential improvements. You can use the preview tab on the Github ci or just go through the translated article on your favourite IDE and make any required improvements. Let me know if you have already done the initial review. I can run through it post that.

**Review by theBstar — CHANGES_REQUESTED** [review]
Thanks for the PR @ayushhagarwal. I can see a lot of potential improvements. You can use the preview tab on the Github ci or just go through the translated article on your favourite IDE and make any required improvements. Let me know if you have already done the initial review. I can run through it post that.

**Comment by ayushhagarwal** [comment]
I have made changes to t he quote and also made my  final review. You can go for your review and then suggest if any more improvements required.

**Comment by theBstar** [comment]
Sorry, @ayushhagarwal I missed your comment completely. You will have to click on the request review once, so one of the reviewers gets notified. I will review this Sunday evening. Thanks anyway

**Pull request closed without merging**